recipes/zen-engine/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "zen-engine" %}
+{% set version = "0.8.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/zen_engine-{{ version }}.tar.gz
+  sha256: e6f550930453b7085025645699a1ec34229aeea10c5f0875618881e285b1d64f
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - python                              # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - crossenv                            # [build_platform != target_platform]
+    - maturin >=0.14,<0.15                # [build_platform != target_platform]
```

Comment:
After this PR gets merged, we need to request an `osx-arm64` package through https://github.com/conda-forge/conda-forge-pinning-feedstock. Example:  https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/3435